test-cases/gutenberg/image.md

```diff
@@ -23,14 +23,14 @@ A site with premium or business plan
 
 ##### TC002
 
-**Insert image from device (cancel)**
+**Insert image from device (stop)**
 
 -   Tap on the Image block placeholder or edit button in the toolbar
 -   Tap option ‘Choose from device’ and choose an image
 -   While the upload is in progress tap on the image thumbnail
 -   You should see the below message:
-![Cancel upload](../resources/cancel-upload.png)
--   Tap cancel
+![Stop upload](../resources/cancel-upload.png)
```

Comment:
The image "../resources/cancel-upload.png" looks outdated in this case. Could you update it with the new version as well?

Reply:
Ok I just realized we have a separate PR for that https://github.com/wordpress-mobile/test-cases/pull/17